### DISCUSSION TITLE
.umirc.ts配置了base、publicPath，访问界面root盒子未渲染

### DISCUSSION THREAD

**Discussion by RunInto** [umijs/umi#11492] [Q&A]
.umiic.ts内容如下：
import { defineConfig } from "umi";

export default defineConfig({
  base: '/public/',
  publicPath: '/public/',

  routes: [
    { path: "/public/index", component: "index" },
  ],
  npmClient: 'pnpm',
  history: {
    type: 'hash',
  },
  
  favicons: [
    '/public/favicon.ico'
  ],
  // proxy: {
  //   '/api': {
  //     'target': 'http://192.168.249.144',
  //     'changeOrigin': true,
  //     'pathRewrite': { '^/api' : '' },
  //   },
  // }
});

访问浏览器如下：
![image](https://github.com/umijs/umi/assets/33835360/5cb52efb-9a9d-4150-806e-48aa1ed18cdc)
![image](https://github.com/umijs/umi/assets/33835360/89bbcea0-b29a-45d2-b31d-404aedce9bc9)
![image](https://github.com/umijs/umi/assets/33835360/3e64d29a-c809-431b-839a-692eb50224cf)
请问，如何才能将root盒子渲染出来？ps(另外我需要配置到nginx的非根目录)

**Comment by fz6m** [comment]
需要给一个最小复现。
